samplers/jaegerremote/example/main.go

```diff
@@ -27,12 +30,19 @@ import (
 )
 
 func main() {
+	// Optional: an implementation of logr.Logger used for demo purposes to catch potential error logs
+	logger := stdr.NewWithOptions(stdlog.New(os.Stderr, "", stdlog.LstdFlags), stdr.Options{LogCaller: stdr.All})
+
+	samplingRefreshInterval := 1 * time.Minute
 	jaegerRemoteSampler := jaegerremote.New(
 		"foo",
-		jaegerremote.WithSamplingServerURL("http://localhost:5778"),
-		jaegerremote.WithSamplingRefreshInterval(10*time.Second), // decrease polling interval to get quicker feedback
+		jaegerremote.WithSamplingServerURL("http://localhost:5778/sampling"),
+		jaegerremote.WithSamplingRefreshInterval(samplingRefreshInterval), // decrease polling interval to get quicker feedback
```

Comment:
This here is not just an upgrade. It changes the refresh interval from 10 seconds to 1 minute, which is the default as well.

https://github.com/open-telemetry/opentelemetry-go-contrib/blob/a1b4b4d348c0ca2dc67834aba13caa2fe5c0a993/samplers/jaegerremote/sampler_remote.go#L37

So we should either:

* Keep the previous value of 10 seconds.
* Remove the option entirely, since this is the default anyway.
* Remove the comment that says we decrease the polling interval, since we don't do that anymore.

I'd tend towards option 1. An example where the refresh interval is refreshed every minute is quite long.

Reply:
👍  I reverted it to the previous value of 10 seconds.